src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/CellService.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.whereabouts.dto.CellMoveDetails
+import uk.gov.justice.digital.hmpps.whereabouts.dto.CellMoveResult
+
+@Service
+class CellService(val prisonApiService: PrisonApiService) {
+  fun makeCellMove(cellMoveDetails: CellMoveDetails): CellMoveResult {
+    return prisonApiService.putCellMove(
```

Comment:
Doesn't do much at the moment but it will orchestrate the creation of case notes and store comments.